app/helpers/mailer_helper.rb

```diff
@@ -56,6 +46,14 @@ def add_reason(text, reason)
     end
   end
 
+  def proposal_attributes(proposal)
```

Comment:
this is the ol' "what happens when we have a dozen clients" comment.

could we delegate this to the client_data class and require it to have a `proposal_attributes` method that is called? That way the client_data class knows how to bootstrap its own client-specific proposal attributes.